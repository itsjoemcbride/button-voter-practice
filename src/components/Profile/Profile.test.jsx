import renderer from "react-test-renderer";
import Profile from "./Profile";

it("renders without crashing", () => {
  const rendered = renderer
    .create(<Profile name="Mark" imgUrl="someImage.jpg" />)
    .toJSON();

  expect(rendered).toMatchSnapshot();
});
