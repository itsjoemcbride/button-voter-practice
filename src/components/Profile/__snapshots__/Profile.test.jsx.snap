// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<section
  className="mx-auto w-48 flex-col items-center"
>
  <h2>
     Photo of 
    Mark
  </h2>
  <img
    alt="Mark"
    className=""
    src="someImage.jpg"
    style={
      Object {
        "height": 200,
        "width": 200,
      }
    }
  />
</section>
`;
